END2ENDTESTS.md

```diff
@@ -0,0 +1,20 @@
+# End-2-End testing guidance
+
+In order to run fully automated end-2-end (e2e) tests Kwenta uses [Synpress](https://github.com/Synthetixio/synpress) (a wrapper around [Cynpress](https://www.cypress.io/)).  
+
+## Constraints 
+The current e2e tests are written to be run on Optimistic Kovan using Chrome as the browser.
+
+## Setup
+- Download and install Google Chrome 
+- Setup a testing wallet on Optimistic Kovan and fund it with plenty of ETH (to pay for gas) and sUSD 
+- Copy the private key of the wallet into the file `TEST-PRIVATEKEY` into the folder `kwenta/kwenta/tests/e2e`
+- Before proceeding to the next step you need to setup environment variables. Navigate to the folder `kwenta/kwenta/tests/e2e` and run the following command `source ./RUN-WITH-SOURCE-synpress-env-KovanOE.sh`
+
+## Run the tests
+
+```bash
```

Comment:
Don't need `bash` here, lets keep it shell agnostic

Reply:
All the other example on README include bash here.. shall I remove altogether from the README file for the other instructions as well?

Reply:
This should be fine then. I didn't realize `"```bash"` was a feature of github markdown